index.js

```diff
@@ -59,6 +59,11 @@ animalToRemove.remove()
 /************************** EVENTS JS MINI CHALLENGE ******************************/
 
 /***** Deliverable 1 *****/
+
+header.addEventListener('click', function(){
+     toggleColor(header) 
+    })
+
```

Comment:
Nicely done! Using arrow functions (ES6 feature), you could also write this solution as shown below. This is the same exact functionality!
`header.addEventListener("click", () => toggleColor(header))`